src/safeds/data/tabular/containers/_column.py

```diff
@@ -1051,10 +1051,12 @@ def stability(self) -> float:
         >>> column.stability()
         0.5
         """
+        import polars as pl
         non_missing = self._series.drop_nulls()
         if non_missing.len() == 0:
             return 1.0  # All non-null values are the same (since there is are none)
-
+        if type(non_missing.dtype)==pl.datatypes.Boolean: 
+            non_missing = non_missing.cast(str)
         mode_count = non_missing.unique_counts().max()
```

Comment:
`value_counts().get_column("count").max()`

Seems to work for boolean as well, which saves us the conversion.